node/pkg/watchers/near/watcher.go

```diff
@@ -249,9 +258,24 @@ func (e *Watcher) runTxProcessor(ctx context.Context) error {
 	}
 }
 
-func (e *Watcher) Run(ctx context.Context) error {
+func (e *Watcher) Run(ctx context.Context) (result error) {
+	defer func() {
```

Comment:
We could build a common function that takes a runnable and returns a runnable that gracefully handles panics.

Reply:
switched to a wrapper